code/backtracking/src/sudoku_solve/sudoku_solveNxN.cpp

```diff
@@ -0,0 +1,96 @@
+
+#include<bits/stdc++.h>
```

Comment:
#1915

Reply:
It's a myth it doesn't increase run time

Reply:
@aryangulati See [this](https://stackoverflow.com/a/31816096/6525260) and [this](https://stackoverflow.com/questions/25311011/how-does-include-bits-stdc-h-work-in-c/33764128#33764128). Regardless of whether it affects performance, it definitely not portable.

Reply:
Waiting will send a proof if u want

Reply:
This repository isn't a programming contest repository. It is a general repository of algorithms for all audiences. Thus, we follow software engineering best practices.

Reply:
Also, more reasons not to use this file: https://www.geeksforgeeks.org/bitsstdc-h-c/

Reply:
If u are looking for this too it's has equal no. Of advantages and disadvantages but for more clarity and some issues suggest by you will change